lib/custom/src/MShop/Service/Provider/Payment/OmniPay.php

```diff
@@ -217,6 +217,20 @@ class OmniPay
 			'required'=> false,
 			'public' => false,
 		),
+
+		'paymenttoken' => array(
```

Comment:
You should move this to the Stripe provider

Reply:
Sorry, my bad. Done